credoai/evaluators/identity_verification.py

```diff
@@ -161,6 +161,12 @@ def _setup(self):
 
         return self
 
+    def _validate_arguments(self):
+        check_data_instance(self.assessment_data, ComparisonData)
+        check_model_instance(self.model, (ComparisonModel, DummyComparisonModel))
+        check_existence(self.assessment_data.pairs, "pairs")
+        return self
+
```

Comment:
The entire method is duplicated, please retain only the version above _setup().

Reply:
Must've been something that came up in merge...?